test/README.md

```diff
@@ -32,12 +32,12 @@ test/presubmit-tests.sh --integration-tests
 _Note that if the tests you are running include `integration tests`, it will
 create a new GKE cluster in project `$PROJECT_ID`, start Knative Serving and
 Eventing system, upload test images to `$KO_DOCKER_REPO`, and run all
-`e2e-*tests.sh` scripts under [`test`](.). After the tests finish, it will
+`e2e-*tests.sh` scripts under [`test`](../contrib/test). After the tests finish, it will
 delete the cluster._
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
`e2e-*tests.sh` scripts under [`test`](../contrib/test). After the tests finish,
it will delete the cluster._
```